tutorial.md

```diff
@@ -0,0 +1,99 @@
+# Tutorial
+This is a tutorial to get started with the Cryosphere Virtual Laboratory (CVL).
+
+## What is CVL
+CVL stands for Cryosphere Virtual Laboratory. It offers users access to the following tools:
```

Comment:
Maybe we could mention that it provides access to the datasets for Polar regions